tools/post-generate.sh

```diff
@@ -0,0 +1,28 @@
+# Ensure the SDK version is consistent with the OpenAPI SHA the CLI is generated from.
+go test -timeout 240s -run TestConsistentDatabricksSdkVersion github.com/databricks/cli/internal/build
```

Comment:
Why run this test before generation? Even if current state is not consistent, generation should fix that, right?